tests/scripts/common_functions.sh

```diff
@@ -156,7 +156,7 @@ function deploy_autotune() {
 	# Check if the cluster_type is minikube., if so deploy prometheus
 	if [ "${cluster_type}" == "minikube" ]; then
 		echo "Installing Prometheus on minikube" >>/dev/stderr
-		setup_prometheus >> ${SETUP_LOG} 2>&1
+		setup_prometheus >> ${AUTOTUNE_SETUP_LOG} 2>&1
```

Comment:
I don't see prometheus cleanup being called anywhere, can you please check and invoke that at required places